README.md

```diff
@@ -1,38 +1,108 @@
-Role Name
-=========
+[![Build Status](https://travis-ci.org/open-io/ansible-role-gridinit.svg?branch=master)](https://travis-ci.org/open-io/ansible-role-gridinit)
+# Ansible role `gridinit`
```

Comment:
ansible-role-openio-gridinit ?

Reply:
Shorter title is better, no ?

Reply:
consistency matters

Reply:
The title of the README, does not change the name of the repo.

Reply:
My bad, yep we can rename the repo with this template 
`ansible-role-openio-XXXX`